dockerfiles/install_common_deps.sh

```diff
@@ -20,6 +20,9 @@ conda install -y python=3.6 numpy
 conda clean -aqy
 rm -rf /opt/miniconda/pkgs
 
+# Add conda location to PATH
+EXPORT PATH=/opt/miniconda/bin:$PATH
```

Comment:
should it be "export" instead of "EXPORT" ?

Reply:
this export statement should go in the higher level dockerfile. Will resubmit the change.